parl/algorithms/torch/ppo.py

```diff
@@ -17,83 +17,125 @@
 import torch.nn as nn
 import torch.nn.functional as F
 import torch.optim as optim
-from torch.distributions import Normal
 from parl.utils.utils import check_model_method
+from torch.distributions import Normal, Categorical
 
 __all__ = ['PPO']
 
 
 class PPO(parl.Algorithm):
-    def __init__(self,
-                 model,
-                 clip_param,
-                 value_loss_coef,
-                 entropy_coef,
-                 initial_lr,
-                 eps=None,
-                 max_grad_norm=None,
-                 use_clipped_value_loss=True):
+    def __init__(self, model, config):
```

Comment:
Please don't replace all hyperparam with config, which will cause great confused of developers of PARL.